packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -24,23 +34,29 @@ export const SafeLocalStorage = {
     value: SafeLocalStorageItems[Key]
   ): void {
     if (isSafe()) {
-      localStorage.setItem(key, JSON.stringify(value))
+      if (typeof value === 'string') {
+        localStorage.setItem(key, value)
+      } else {
+        localStorage.setItem(key, JSON.stringify(value))
+      }
```

Comment:
We need to keep parsing string to JSON string, otherwise it will fail when parsing back